src/funnels/Swapper.sol

```diff
@@ -38,15 +38,16 @@ contract Swapper {
     address   public immutable buffer; // Contract from which the GEM to sell is pulled and to which the bought GEM is pushed
 
     struct PairLimit {
-        uint64  hop; // Cooldown one has to wait between each src to dst swap
-        uint64  zzz; // Timestamp of the last src to dst swap
-        uint128 cap; // Maximum amount of src token that can be swapped each hop for a src->dst pair
+        uint96  cap; // Maximum amount of src token that can be swapped each hop for a src->dst pair
+        uint32  hop; // Cooldown period it has to wait between each src to dst swap
```

Comment:
What do you think about changing `hop` to `era` in Swapper and Depositor? `hop` is usually an interval between operations, while `era` is a time interval where several events can happen.
I'd also rename `zzz`, as that indicates a sleeping time, maybe to `beg` to indicate the beginning?
With these renamings we also void confusion with the automation contracts, which have `hop` and `zzz`, but with the intended meanin.

Reply:
We also need to update the documentation of the variables, as it still just assumes one swap in the beginning of the `hop` period everywhere.

Reply:
+1 to using `era` and whatever else for the running timestamp (`beg` is fine).

Reply:
Changed, I used `end` changing a bit the logic (which I think is more efficient).